akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -225,4 +224,17 @@ object Sink extends SinkApply {
   def actorSubscriber[T](props: Props): Sink[T, ActorRef] =
     new Sink(new ActorSubscriberSink(props, none, shape("ActorSubscriberSink")))
 
+  /**
+   * Creates a synchronous (Java 6 compatible) Sink that writes incoming [[ByteString]] elements to the given file.
+   *
+   *
+   * Materializes a [[Future]] that will be completed with the size of the file (in bytes) at the streams completion.
+   *
+   * This source is backed by an Actor which will use the dedicated `akka.stream.file-io-dispatcher`,
+   * unless configured otherwise by using [[ActorOperationAttributes]].
+   */
+  def synchronousFile(f: File, append: Boolean = false): Sink[ByteString, Future[Long]] =
```

Comment:
Document that this is not only synchronous but blocking. I know it is clear for us, but better be safe.